.github/workflows/ci.yml

```diff
@@ -2,6 +2,8 @@ name: CI
 
 on:
   pull_request:
+    branches:
+      - main
```

Comment:
I can’t say why exactly, but workflows were not triggered since yesterday and until i ve added branch name. 
also I don’t see any other maintained branches

Reply:
It can be useful for short-lived branches, e.g. if I want to make a PR into this PR